mempool/src/config.rs

```diff
@@ -72,3 +70,17 @@ pub const MAX_ORPHAN_ACCOUNT_GAP: u64 = 2;
 pub const FUTURE_TIMELOCK_TOLERANCE: Duration = Duration::from_secs(5 * 60);
 
 pub const FUTURE_TIMELOCK_TOLERANCE_BLOCKS: BlockDistance = BlockDistance::new(5);
+
+// 10^-7 of a coin
+make_config_setting!(MinTxRelayFeePerByte, Amount, Amount::from_atoms(10_000));
+
+#[derive(Debug, Clone, Default)]
+pub struct MempoolConfig {
+    pub min_tx_relay_fee_per_byte: MinTxRelayFeePerByte,
```

Comment:
Would be more accurate to rename both the field name and the type to state `feerate` rather than `fee`.